### PR TITLE
New package: wayshot-1.2.2

### PR DESCRIPTION
**Pull request by akhiljalagam**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### New package
- This new package conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements): **YES**

### CODE REVIEW

**Comment by akhiljalagam** [comment]
oops. found a duplicate https://github.com/void-linux/void-packages/pull/36000

File: srcpkgs/wayshot/template [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Template file for 'wayshot'
+pkgname=wayshot
+version=1.1.9
+revision=1
+build_style=cargo
+makedepends="scdoc"
```
Comment (classabbyamp) [inline srcpkgs/wayshot/template:6]: ahould be hostmakedepends, and you need to install the manpages it generates

Comment (akhiljalagam) [reply srcpkgs/wayshot/template:6]: `wayshot.1.scd` is not available in the releases tar.gz file.

Comment (akhiljalagam) [reply srcpkgs/wayshot/template:6]: but its showing the repo https://github.com/waycrate/wayshot/tree/main/docs

Comment (classabbyamp) [reply srcpkgs/wayshot/template]: ah, looks like it was added after the release

**Comment by Shinyzenith** [comment]
Hi! apologies for the issues. I have tagged a release with the scdoc man pages ❤️

**Comment by Shinyzenith** [comment]
Another thing that I should mention, the build.rs file compiles the man pages automatically if the scdoc binary is found on the build container 😁.

**Comment by akhiljalagam** [comment]
why it's taking this much time to complete the build?  @classabbyamp

**Comment by Shinyzenith** [comment]
> why it's taking this much time to complete the build? @classabbyamp

I'm afraid I know why....I'll tag a release with fix when I get home...my apologies once again. Basically I did a scdoc check in build.rs and it inherited the stdio of parent process and kept waiting for input.

**Comment by Shinyzenith** [comment]
https://github.com/waycrate/wayshot/releases/tag/1.2.1 fixed!

**Comment by akhiljalagam** [comment]
> https://github.com/waycrate/wayshot/releases/tag/1.2.1 fixed!

thanks. it worked finally. but the man pages are installed only when used shell redirection. I think the way void build rust packages is different.

**Comment by classabbyamp** [comment]
> the man pages are installed only when used shell redirection. I think the way void build rust packages is different.

you can just

**Comment by classabbyamp** [comment]
> the man pages are installed only when used shell redirection

```
$ ./xbps-src build wayshot
...
$ ls masterdir/builddir/wayshot-1.2.1/docs
assets  wayshot.1.gz  wayshot.1.scd  wayshot.7.gz  wayshot.7.scd
$ file masterdir/builddir/wayshot-1.2.1/docs/wayshot.1.gz 
masterdir/builddir/wayshot-1.2.1/docs/wayshot.1.gz: troff or preprocessor input, ASCII text
```
@Shinyzenith it's still broken, you're writing uncompressed manpages but calling them gzipped

@akhiljalagam you shouldn't need to build them with scdoc yourself, they build properly because `build.rs` always runs for the host, not target, on cross

**Comment by akhiljalagam** [comment]
Understood. I will remove manual scdoc cmds then.  
@Shinyzenith once you do the fix, I can release with the required changes.

**Comment by Shinyzenith** [comment]
> 

Okay! I'll export them as *.1 files and gzip them! 
Will do it asap.

**Comment by Shinyzenith** [comment]
Done!

**Comment by akhiljalagam** [comment]
> Done!

release 1.2.2 also not installing man pages automatically.

**Comment by Shinyzenith** [comment]
> > Done!
> 
> release 1.2.2 also not installing man pages automatically.

We don't install the man pages, we just transpire and gzip them programmatically. Isn't that supposed to he done on the package side ?

**Comment by akhiljalagam** [comment]
@Shinyzenith oh. got it now. thanks.

**Comment by akhiljalagam** [comment]
done. tested 1.2.2. working now.

**Comment by akhiljalagam** [comment]
buiild failed for musl with fololowing error
`ERROR: [trans] failed to download `libllvm12-12.0.1_3' signature from `https://repo-ci.voidlinux.org/current/musl': Not Found`

**Comment by icp1994** [comment]
Your template almost works, just remove the `scdoc` commands from the `post_install` - it should be:
```
post_install() {
	vlicense LICENSE
	vman docs/wayshot.1.gz
	vman docs/wayshot.7.gz
}
```

File: srcpkgs/wayshot/template [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+license="BSD-2-Clause"
+homepage="https://github.com/waycrate/wayshot"
+changelog="https://github.com/waycrate/wayshot/releases"
+distfiles="https://github.com/waycrate/wayshot/archive/refs/tags/${version}.tar.gz"
+checksum=6f94150cbd7984d941f53680a5116c5b3ee27ab7650e88dc8c7d09742410b049
+
+post_install() {
+	vlicense LICENSE
+	vman ./docs/wayshot.1.gz
+	vman ./docs/wayshot.7.gz
```
Comment (classabbyamp) [inline srcpkgs/wayshot/template:18]:
```suggestion
	vman docs/wayshot.1.gz
	vman docs/wayshot.7.gz
```

**Comment by Shinyzenith** [comment]
Hi! just curious, is anything blocking this that's supposed to be fixed upstream?

**Comment by akhiljalagam** [comment]
> Hi! just curious, is anything blocking this that's supposed to be fixed upstream?

void linux needs more volunteers

**Pull request closed without merging**